code/modules/modular_computers/file_system/programs/generic/ntsl2_interpreter.dm

```diff
@@ -57,9 +160,9 @@
 		. = TRUE
 
 	if(.)
-		SSnanoui.update_uis(NM)
-
+		SSnanoui.update_uis(NM)*/
 
+/*
```

Comment:
Remove dead code instead of commenting it out

Reply:
Deadcode: Removed